cmd/root.go

```diff
@@ -43,7 +44,7 @@ func newRootCmd() *cobra.Command {
 	rootCmd.AddCommand(versionCmd)
 	rootCmd.AddCommand(get.CreateGetCommand())
 	rootCmd.AddCommand(update.CreateUpdateCommand())
-	rootCmd.AddCommand(register.RegisterCommand())
+	rootCmd.AddCommand(register.FlyteRegisterCommand())
```

Comment:
Why not register command? I guess because it has a weird meaning.
What about `RemoteRegisterCommand`